device-services/grove-c/README.md

```diff
@@ -0,0 +1,8 @@
+# demo-grove-pi
+Contain scripts to pull the images and run them for devkit based on Raspberry PI
```

Comment:
Change to:  This example contains the instructions and scripts to connect the Grove Sensor Kit to a Raspberry Pi 3 and use EdgeX to get data from the kit's sensors and command the kit's devices.

This example uses the EdgeX Geneva release.

Reply:
Should this example be upgraded to use Hanoi?

Reply:
> Should this example be upgraded to use Hanoi?

This revision of docker-compose is tested with Geneva release. There will be a separate PR with the required changes in to use Hanoi.